install.sh

```diff
@@ -4,37 +4,54 @@
 
 #This assumes all is installed under /opt/HardwareCheckout
 
-APP_PATH=/opt/hardware
-GIT_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+GITREPO=https://github.com/linted/HardwareCheckout.git
```

Comment:
Git repo should be from wherever it was cloned. The way I did it in master is correct. That allows for forks of the repo to still work correctly.

Reply:
Added